docs/vz330_mellow/electronics/Touchscreen.md

```diff
@@ -3,129 +3,144 @@ layout: default
+    ![Assembly3](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly3.jpg)
 
+4. Insert the 2 M4 x 8mm screws into the mounting hinge, and loosely apply 2 M4 T-nuts to these screws  
 
-## Klipperscreen
+    ![Assembly4](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly4.jpg)
 
-We're going to install Klipperscreen so we can use our touchscreen.
+5. Fasten the mounting hinge to the printer frame using the 2 M4 screws and corresponding T-nuts previously inserted.
 
-First we're going to SSH into the Pi with Putty and we're going to install KIAUH
+    These screws don't need to be overly tight, as you may need to adjust the display to ensure smooth cooperation with the enclosure panels.
 
-```bash
-sudo apt-get update && sudo apt-get install git -y
-```
-```bash
-cd ~ && git clone https://github.com/dw-0/kiauh.git
-```
-```bash
-./kiauh/kiauh.sh
-```
+6. Loosely attach the display assembly to the mounting hinge using 1 M4 x 25mm screw and one M4 hex nut
 
-Then Press 1 for install and hit enter
+    Don't overtighten these screws. The screw acts as a hinge and needs room to move ever so slightly.
 
-![kiauh](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/KIAUH.PNG)
+    If you have not wired up the DSI cable yet, then dismount the display from the mounting hinge again after confirming the hinge works correctly.
 
-Press 5 for Klipperscreen and hit enter. This might take a while to install.
+    ![Assembly5](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Assembly5.jpg)
 
-![klipperscreen](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Option5.PNG)
+## Wiring
 
-### Touchscreen issue's
+Wiring up the display requires careful **attention to detail**, as the direction of the flat-flex cable is of utmost importance.
 
-If you're touchscreen isn't working SSH into your Pi with Putty and run:
+The DSI cable runs between the LCD display Core board and the DSI connector on the Raspberry Pi. The USB-C port on the LCD display PCB does not need connecting up when using the DSI cable.
 
-```bash
-sudo nano /boot/config.txt
-```
-![Touchscreen](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/Touchscreen.PNG)
+DSI cable highlighted in red:
 
-Then scroll down and change the highlited line From: 
-```bash
-dtoverlay=vc4-kms-v3d
-```
+![DSI cable from kit, highlighted in red](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/dsi-cable-highlighted.jpg)
 
-To: 
+### Display
 
-```bash
-dtoverlay=vc4-fkms-v3d
-```
-![TouchscreenEdit](../../assets/images/manual/vz235_printed/electronics/Firmware/Touchscreen/TouchscreenEdit.PNG)
+1. Lie the display assembly down on a flat surface, back side up.
 
+    ![Display assembly, face down]()
 
-Press CTRL+X and then Y to save and exit. Now reboot your Pi and You're screen should be fully working.
```

Comment:
Also has to stay in to solve this common issue.

Reply:
I've written this section on the KlipperScreen docs. Using a link to that manual's instructions is probably beneficial to both our conciseness, and future-readiness in the event this issue gets fixed. 
https://klipperscreen.readthedocs.io/en/latest/Troubleshooting/Touch_issues/#touch-not-working-on-raspberryos-and-derivatives

Reply:
Admittedly I did totally forget to add that part when I made my initial refactors.

This is just a draft, and is still going to change. I pushed this to the branch so I could work on it from the workshop too :)

Reply:
Then we have to put the link in here somewhere. i suggest moving the ###touchscreen issues to the bottom with the link there and saying in case of issue's with the touchscreen go here .... for instructions how to fix it. That's better then just completely getting rid of it

Reply:
Yeah that's fine. My intent was never to completely get rid of it. I simply forgot to add it back in 🙈